src/basic/process-util.c

```diff
@@ -582,6 +582,14 @@ int pidref_get_uid(const PidRef *pid, uid_t *ret) {
         if (pidref_is_remote(pid))
                 return -EREMOTE;
 
+        if (pid->fd >= 0) {
+                r = pidfd_get_uid(pid->fd, ret);
+                if (r != -EOPNOTSUPP)
+                        return r;
+        }
+
+        uid_t uid;
+
```

Comment:
i'd remove this empty line, i.e. always declare variables that are needed for some immediately following functional call actually immediately before it.